pages/linux/expect.md

```diff
@@ -0,0 +1,16 @@
+# expect
+
+> Programmed dialogue with interactive programs.
+> More intermation <https://linux.die.net/man/1/expect>.
+
+- Execute expect scripts from file:
```

Comment:
Perhaps this will read better?
```suggestion
- Execute an expect script from a file:
```

Also, could `{{file}}` on the following line be `{{path/to/file}}` please?